### PR TITLE
Store and checkout page mvp complete

### PR DESCRIPTION
**Pull request by quyencodes**
- Completed CLRF to LF conversion and change to seed.sh for products and skus
- Completed backend for checkout - http delete request completed
- Complete 90% of checkout page, front-end complete, need to make successful delete request for removing item from a cart
- Functionality of checkout page 50% done, css and general logic completed, need to finish http delete request and fix
 minor bugs

### CODE REVIEW

File: client/src/pages/Checkout/index.jsx [hunk 1]
```diff
@@ -1,59 +1,89 @@
...
 import './checkout.css'
 
 export default function Checkout() {
   const [cart, setCart] = useState(0)
   const [subtotal, setSubtotal] = useState(0)
   const [tax, setTax] = useState(0)
-  const [total, setTotal] = useState(0)
 
   useEffect(() => {
     axios
```
Comment (keewook2) [inline client/src/pages/Checkout/index.jsx:13]: I recommend to make a fetch function outside of useEffect and call that function inside of useEffect, in that case when you need another fetch request (such as when cart content is changed, you gonna re-fetch from api), you can reuse fetch function

File: server/data/seed.sh [hunk 2]
```diff
@@ -17,8 +17,6 @@ psql -U postgres -d blueocean -c "\copy fans FROM './fans.csv' DELIMITER ',' CSV
 echo "Importing members data..."
 psql -U postgres -d blueocean -c "\copy members FROM './members.csv' DELIMITER ',' CSV HEADER;"
```
Comment (keewook2) [inline server/data/seed.sh:19]: make sure remove line 3 (exporting PGPASSWORD = #$@$#!@$#), that's sensitive information.

**Review by keewook2 — APPROVED** [review]
lgtm

File: client/src/pages/Checkout/Items.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import axios from 'axios'
+import './items.css'
+
+export default function Items({ cart, setCart }) {
+  console.log("this is what's in the cart", cart)
```
[the remark this answers is not in the document]
Comment (quyencodes) [reply client/src/pages/Checkout/Items.jsx:5]: thank you, done.

**Pull request merged**